vonage/logs/vonage_mapper.yaml

```diff
@@ -0,0 +1,148 @@
+id: REPLACE_WITH_APP_ID
```

Comment:
this `/logs` folder should be moved under `assets`, so these files would be at `vonage/assets/logs/vonage_mapper.yml` for example.